.kitchen.yml

```diff
@@ -128,3 +131,14 @@ suites:
     includes:
       - windows-2012r2
       - windows-2016
+
+  - name: rogue_server
+    includes:
+      - ubuntu-18.04
+    attributes:
+      consul:
+        is_rogue_server: true
```

Comment:
I think `is_decommissioned_server` might be a better name. I don't think new comers will understand what we mean by rogue.

Reply:
Got it, instead of is_rogue_server, I renamed to is_unexpected_server to be even more general.